Airports/EGBB/Fixes.txt

```diff
@@ -45,14 +45,12 @@ BBSTUD N052.16.11.588 W001.55.29.664
 BBWARW N052.15.37.108 W001.36.48.480
 BBINTE N052.21.11.131 W001.52.56.741
 BBFRAN N052.25.19.752 W002.03.13.078
-BB33C N052.28.28.387 W001.46.20.090
-BB33D N052.29.32.814 W001.43.34.694
-BB33B N052.26.32.419 W001.40.21.733
-BB33A N052.25.18.789 W001.42.45.688
-BB15C N052.26.06.649 W001.43.40.820
-BB15D N052.27.12.916 W001.41.04.613
-BB15B N052.30.18.833 W001.44.23.700
-BB15A N052.29.16.247 W001.47.02.970
+BB33UC N052.28.28.387 W001.46.20.090
+BB33CD N052.29.32.814 W001.43.34.694
+BB33DB N052.26.32.419 W001.40.21.733
+BB15UC N052.26.06.649 W001.43.40.820
+BB15CD N052.27.12.916 W001.41.04.613
+BB15DB N052.30.18.833 W001.44.23.700
 BB33MDN N052.28.10.185 W001.42.05.032
 BB33MDS N052.26.26.469 W001.47.54.179
 BB33OH N052.26.36.584 W001.44.11.532
```

Comment:
There's no BB15OH fix. This might have been the case before but would be good to add one if we can get it done tonight

Reply:
Haha I thought so too, but it is there... just not in order with everything else.
https://github.com/VATSIM-UK/UK-Sector-File/blob/issue-3608/Airports/EGBB/Fixes.txt#L57

Reply:
It's hiding :(

Reply:
I did wonder how I would have missed that the first time round